### PR TITLE
Adds emergency meetings during april fools

### PR DESCRIPTION
**Pull request by MrDoomBringer**
## About The Pull Request
# [For the benefit of the maintainers, April Fools PRs will be labeled and closed. Key Holders are still able to test merge PRs that are closed, so this makes no difference to you, the author, assuming you were able to find a Key Holder to merge this PR in the first place. If you need to make updates to this PR, you can simply add to the branch, and when the TM updates it will use those changes, even if they do not show up to the PR.](https://streamable.com/jd8jd2)

## Why It's Good For The Game
For the benefit of the maintainers, April Fools PRs will be labeled and closed. Key Holders are still able to test merge PRs that are closed, so this makes no difference to you, the author, assuming you were able to find a Key Holder to merge this PR in the first place. If you need to make updates to this PR, you can simply add to the branch, and when the TM updates it will use those changes, even if they do not show up to the PR.

## Changelog
:cl:
add: The captain can now sometimes call emergency meetings using the communications console! Hopefully the crew will be able to sus out any imposters who might be among them!
/:cl:

### CODE REVIEW

**Comment by MrDoomBringer** [comment]
For the benefit of the maintainers, April Fools PRs will be labeled and closed. Key Holders are still able to test merge PRs that are closed, so this makes no difference to you, the author, assuming you were able to find a Key Holder to merge this PR in the first place. If you need to make updates to this PR, you can simply add to the branch, and when the TM updates it will use those changes, even if they do not show up to the PR.

**Comment by Ryll-Ryll** [comment]
oh god no

**Comment by tralezab** [comment]
holy fuck

**Comment by tralezab** [comment]
>

**Comment by tralezab** [comment]
>

**Comment by tralezab** [comment]
>

**Comment by optimumtact** [comment]
For the benefit of the maintainers, April Fools PRs will be labeled and closed. Key Holders are still able to test merge PRs that are closed, so this makes no difference to you, the author, assuming you were able to find a Key Holder to merge this PR in the first place. If you need to make updates to this PR, you can simply add to the branch, and when the TM updates it will use those changes, even if they do not show up to the PR.

**Comment by tralezab** [comment]
>

**Comment by MrDoomBringer** [comment]
the whole crew has made it in

**Comment by MrDoomBringer** [comment]
can be merged in if oranges so desires, the headadmins have annointed/blessed this PR etc

**Comment by Serxule** [comment]
TERRORISM TIME :)

**Comment by Mothblocks** [comment]
> can be merged in if oranges so desires, the headadmins have annointed/blessed this PR etc

if you say so

File: code/__HELPERS/priority_announce.dm
Comment (Mothblocks) [inline code/__HELPERS/priority_announce.dm:63]: im working off the assumption you're expecting this to get merged

https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names

File: code/__HELPERS/priority_announce.dm
Comment (Mothblocks) [inline code/__HELPERS/priority_announce.dm:64]: https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names

File: code/__HELPERS/priority_announce.dm
Comment (Mothblocks) [inline code/__HELPERS/priority_announce.dm:65]: early continue

File: code/__HELPERS/priority_announce.dm
Comment (Mothblocks) [inline code/__HELPERS/priority_announce.dm:69]:
```suggestion
			addtimer(CALLBACK(M, /mob/.proc/clear_fullscreen, "emergency_meeting"), 3 SECONDS)
```

File: code/controllers/subsystem/communications.dm
Comment (Mothblocks) [inline code/controllers/subsystem/communications.dm:10]: cooldown defines

File: code/controllers/subsystem/communications.dm
Comment (Mothblocks) [inline code/controllers/subsystem/communications.dm:46]: just use return

File: code/game/machinery/computer/communications.dm
Comment (Mothblocks) [inline code/game/machinery/computer/communications.dm:167]: silicons are gonna be able to call emergency meetings?

File: code/game/machinery/computer/communications.dm
Comment (Mothblocks) [inline code/game/machinery/computer/communications.dm:300]: wtf

**Review by Mothblocks — CHANGES_REQUESTED** [review]
im assuming you plan on getting this merged

File: code/__HELPERS/priority_announce.dm
Comment (Dorsisdwarf) [inline code/__HELPERS/priority_announce.dm:74]: Doesn't the way this while loop is set up mean that the room will fill with trash mobs and not the crew, then abort before having actually brought all the players (Who are the people we actually want to dump in a room with the captain and fifty bees)?

File: code/game/machinery/computer/communications.dm
Comment (MrDoomBringer) [reply code/game/machinery/computer/communications.dm:167]: yes direct blame to athath

File: code/game/machinery/computer/communications.dm
Comment (MrDoomBringer) [reply code/game/machinery/computer/communications.dm:300]: >:)

File: code/__HELPERS/priority_announce.dm
Comment (MrDoomBringer) [reply code/__HELPERS/priority_announce.dm:74]: nah this while loop is just for finding a suitable turf to deploy the particular mob to, the outer for-loop handles getting every single mob in there (beepsky, ian, ghosts, corpses, and the living are all valuable crewmembers!)

File: code/game/machinery/computer/communications.dm
Comment (MrDoomBringer) [reply code/game/machinery/computer/communications.dm:167]: ai will save the round from the imposter

File: code/controllers/subsystem/communications.dm
Comment (Mothblocks) [reply code/controllers/subsystem/communications.dm:11]: it still doesnt use the cooldown defines

Comment (MrDoomBringer) [reply code/controllers/subsystem/communications.dm:11]: do you mean the cooldown macros? i set it to use a define for the cooldown time

**Comment by stylemistake** [comment]
Needs more polish.

**Comment by hermitec** [comment]
When the clown is traitorous! 
image

**Review by Mothblocks — APPROVED** [review]
@optimumtact ball is in your court now

**Comment by maxymax13** [comment]
image
license memes in shambles

**Pull request merged**
